app/messaging/process-file-message.js

```diff
@@ -15,6 +16,7 @@ const processFileMessage = async (message, receiver) => {
     await receiver.completeMessage(message)
   } catch (err) {
     console.error('Unable to process message:', err)
+    sendFailureEvent(message.OutputFileName, err)
```

Comment:
I don't think this will capture all connection issues.

This function runs once a message is received by Service Bus.  If there was a service bus connection issue then it wouldn't get as far as this try/catch.

I think you may need to hook into the service bus subscription lifecycle for that.

This should catch storage connection issues though.

Reply:
added an additional catch at service bus level - didn't know I could do that! 😃 
/